admin/model/extension/payment/razorpay.php

```diff
@@ -203,32 +203,32 @@ public function enablePlan($entity_id)
             $frequency = "yearly";
         }
 
-      $data = array(
-                    'plan_name'=>$planData['plan_name'],
-                    'plan_entity_id'=>$entity_id,
-                    'status'=>1,
-                    'price'=>$planData['plan_bill_amount'],
-                    'frequency'=>$frequency,
-                    'duration'=>$planData['plan_frequency'],
-                    'cycle'=>$planData['plan_bill_cycle'],
-                    'trial_status'=>0,
-                    'trial_price'=>$planData['plan_trial'],
-                    'trial_frequency'=>'day',
-                    'trial_duration'=>0,
-                    'trial_cycle'=>0,
-                    'product_id'=>$planData['opencart_product_id'],
-                    'customer_group_id'=>$this->config->get('config_customer_group_id'),
-                    'languages'=>$this->model_localisation_language->getLanguages()
-      );
+        $data = array(
+            'plan_name'=>$planData['plan_name'],
+            'plan_entity_id'=>$entity_id,
+            'status'=>1,
+            'price'=>$planData['plan_bill_amount'],
+            'frequency'=>$frequency,
+            'duration'=>$planData['plan_frequency'],
+            'cycle'=>$planData['plan_bill_cycle'],
+            'trial_status'=>0,
+            'trial_price'=>$planData['plan_trial'],
+            'trial_frequency'=>'day',
+            'trial_duration'=>0,
+            'trial_cycle'=>0,
+            'product_id'=>$planData['opencart_product_id'],
+            'customer_group_id'=>$this->config->get('config_customer_group_id'),
+            'languages'=>$this->model_localisation_language->getLanguages()
+        );
         $this->addRecurring($data);
 
         // update status
-        $this->db->query("UPDATE " . DB_PREFIX . "razorpay_plans SET plan_status = '" . 1 . "'WHERE entity_id = '" .$entity_id . "'");
+        $this->db->query("UPDATE " . DB_PREFIX . "razorpay_plans SET plan_status = '" . 1 . "'WHERE entity_id = '" .(int)$entity_id . "'");
```

Comment:
@sonyy-m follow a common convention for leaving white space before and after concatenation operator